patches/chromium/mas_no_private_api.patch

```diff
@@ -6,655 +6,23 @@ Subject: mas_no_private_api.patch
-+#endif
-       classic_discovery_manager_(
-           BluetoothDiscoveryManagerMac::CreateClassic(this)),
-       low_energy_discovery_manager_(
-@@ -336,8 +340,12 @@ CBCentralManagerState GetCBManagerState(CBCentralManager* manager) {
- }
- 
- bool BluetoothAdapterMac::SetPoweredImpl(bool powered) {
-+#ifndef MAS_BUILD
-   power_state_function_.Run(base::strict_cast<int>(powered));
-   return true;
-+#else
-+  return false;
-+#endif
- }
- 
- void BluetoothAdapterMac::RemovePairingDelegateInternal(
-diff --git a/media/audio/BUILD.gn b/media/audio/BUILD.gn
-index d56fc21776812d36870767ea04a25d72287a87eb..c3b0a303693439947bbd804df69adf5a23e692a1 100644
---- a/media/audio/BUILD.gn
-+++ b/media/audio/BUILD.gn
-@@ -177,6 +177,12 @@ source_set("audio") {
-       "mac/scoped_audio_unit.cc",
-       "mac/scoped_audio_unit.h",
-     ]
-+    if (is_mas_build) {
-+      sources -= [
-+        "mac/coreaudio_dispatch_override.cc",
-+        "mac/coreaudio_dispatch_override.h",
-+      ]
-+    }
-     frameworks = [
-       "AudioToolbox.framework",
-       "AudioUnit.framework",
-diff --git a/media/audio/mac/audio_manager_mac.cc b/media/audio/mac/audio_manager_mac.cc
-index a994f74bb68d1e57ffa787e159f0a6c69e7b6953..1d84f84b59b2bae75c10c00da730022eca52d258 100644
---- a/media/audio/mac/audio_manager_mac.cc
-+++ b/media/audio/mac/audio_manager_mac.cc
-@@ -882,7 +882,7 @@ AudioParameters AudioManagerMac::GetPreferredOutputStreamParameters(
- 
- void AudioManagerMac::InitializeOnAudioThread() {
-   DCHECK(GetTaskRunner()->BelongsToCurrentThread());
--  InitializeCoreAudioDispatchOverride();
-+  // InitializeCoreAudioDispatchOverride();
-   power_observer_.reset(new AudioPowerObserver());
- }
- 
 diff --git a/net/dns/dns_config_service_posix.cc b/net/dns/dns_config_service_posix.cc
-index bd838cbf757deed871a166d5c3205a6619c87d68..efedc7f089ad9f995d52ca0c50378b99237ae6cf 100644
+index b50f468c7666150ed501566d36ebe4e1fd32227f..4df4951e35aff1e02e5cd831211866279f44fedc 100644
 --- a/net/dns/dns_config_service_posix.cc
 +++ b/net/dns/dns_config_service_posix.cc
-@@ -248,6 +248,7 @@ class DnsConfigServicePosix::Watcher {
+@@ -234,6 +234,7 @@ class DnsConfigServicePosix::Watcher : public DnsConfigService::Watcher {
+     CheckOnCorrectSequence();
  
-   bool Watch() {
      bool success = true;
-+#ifndef MAS_BUILD
++    #ifndef MAS_BUILD
      if (!config_watcher_.Watch(base::BindRepeating(&Watcher::OnConfigChanged,
                                                     base::Unretained(this)))) {
        LOG(ERROR) << "DNS config watch failed to start.";
-@@ -270,6 +271,7 @@ class DnsConfigServicePosix::Watcher {
-                                 DNS_CONFIG_WATCH_MAX);
+@@ -251,6 +252,7 @@ class DnsConfigServicePosix::Watcher : public DnsConfigService::Watcher {
+       success = false;
      }
  #endif  // !defined(OS_ANDROID) && !defined(OS_IOS)
 +#endif
      return success;
    }
  
-diff --git a/sandbox/mac/sandbox_compiler.cc b/sandbox/mac/sandbox_compiler.cc
```

Comment:
these changes don't seem right, was this intentional?

Reply:
Oh that's odd - was that patch deleted? 🤔 I don't think that's right and is likely the result of a bad/incomplete sync on my end, will try to fix it now. Thanks for the catch, @miniak

Reply:
Found the problem commit that changed this, fixing now: https://github.com/electron/electron/pull/27280/commits/7214406ecbe64648605778b74fddc3e06c21c6c9